sapi/phpdbg/phpdbg_info.c

```diff
@@ -403,12 +403,16 @@ PHPDBG_INFO(classes) /* {{{ */
 		phpdbg_print_class_name(ce);
 
 		if (ce->parent) {
-			zend_class_entry *pce;
-			pce = ce->parent;
-			do {
-				phpdbg_out("|-------- ");
-				phpdbg_print_class_name(pce);
-			} while ((pce = pce->parent));
+			if (ce->ce_flags & ZEND_ACC_LINKED) {
+				zend_class_entry *pce;
```

Comment:
I thought you liked 1 line declaration/assignation :-P. kidding.

Reply:
I very much do :p, I just didn't pay attention to this (this is pre-existing code)